src/screen/dashboard.rs

```diff
@@ -79,12 +79,16 @@ impl Dashboard {
                     self.panes.resize(&split, ratio);
                     self.last_changed = Some(Instant::now());
                 }
-                pane::Message::PaneDragged(pane_grid::DragEvent::Dropped {
-                    pane,
-                    target,
-                    region,
-                }) => {
-                    self.panes.split_with(&target, &pane, region);
+                pane::Message::PaneDragged(pane_grid::DragEvent::Dropped { pane, target }) => {
+                    match target {
+                        pane_grid::Target::Edge(edge) => {
+                            self.panes.move_to_edge(&pane, edge);
+                        }
+                        pane_grid::Target::Pane(target, region) => {
+                            self.panes.split_with(&target, &pane, region);
+                        }
+                    }
```

Comment:
You can use helper method `self.panes.drop(&pane, target);` instead

Reply:
Nice :)

Reply:
Fixed in [9c77b7b](https://github.com/squidowl/halloy/pull/114/commits/9c77b7be7ec6d0c7901c1ad5616f051437c2211e)